monitors/test/ContractMonitor.js

```diff
@@ -207,6 +207,10 @@ contract("ContractMonitor.js", function(accounts) {
     assert.isTrue(lastSpyLogIncludes(spy, "280.00%")); // expected collateralization ratio of 280%
     assert.isTrue(lastSpyLogIncludes(spy, "140.00")); // liquidated collateral amount of 150 - 10
     assert.isTrue(lastSpyLogIncludes(spy, "150.00")); // locked collateral amount of 150
+    assert.isTrue(lastSpyLogIncludes(spy, "50.00")); // tokens liquidated
+    assert.isTrue(lastSpyLogIncludes(spy, "150.00%")); // cr requirement %
+    assert.isTrue(lastSpyLogIncludes(spy, "1.000")); // estimated price at liquidation time
```

Comment:
No idea why, but a price of `toWei("1")` appears to be formatted by `createFormatFunction` as `1.000`, but shouldn't this only be 2 decimals?

Reply:
right now the formatting utils will use the `minPrecision` if the number is less than _or equal to_. 1. This should perhaps be changed to greater than or equal to on [this](https://github.com/UMAprotocol/protocol/blob/1aee1c935463dcc1e2bb7964fa4f82adf1abd20c/common/FormattingUtils.js#L51) line in the formatting util.

Reply:
yeah let me make that change